XYZDrivers/src/main/java/com/xyzdrivers/services/SQLService.java

```diff
@@ -363,19 +367,18 @@ public void insert(String table, Object[] values) throws SQLException
      */
     public void insert(String table, Object[][] values) throws SQLException
     {
-        for (int j = 0; j < values.length; j++)
-        {
+        for (Object[] value : values) {
             //prepare statement query "INSERT INTO table VALUES (?, ...)"
             String insertQuery = "INSERT INTO "+table+" VALUES (";
-            for (int i = 0; i < values[j].length; i++)
-            {
-                if (i == values[j].length-1)
+            for (int i = 0; i < value.length; i++) {
+                if (i == value.length - 1) {
```

Comment:
There is no need for the if else statement in the block. If you just loop through until value.length - 1 then perform the insertQuery += ("?)"); you can reduce the cyclometric complexity. i.e.

for (int i = 0; i < value.length - 1; i++) {
    insertQuery += ("?, ");
}
insertQuery += ("?)");

Reply:
This isn't my code - I just hit alt-enter and let NetBeans simplify. @GeaRSiX 

Otherwise, you're absolutely right.